sass/base/components/navigation/feature-nav-open.twig

```diff
@@ -1,5 +1,5 @@
 <div class="header__inner-wrapper">
-  <nav style="top:-10px" role="navigation" aria-labelledby="block-campaign-base-main-menu-menu" id="block-campaign-base-main-menu" class="contextual-region block block-menu navigation menu--main crNavigation-processed show">          
+  <nav style="top:-10px" role="navigation" aria-labelledby="block-campaign-base-main-menu-menu" id="block-campaign-base-main-menu" class="contextual-region block block-menu navigation menu--main feature-main-nav crNavigation-processed show">          
```

Comment:
oh I changed this file to add a specific classname to the feature nav as it was conflicting with the main nav. The styling of the feature nav and how it sits in the pattern-lab needs to be looked at separately I think.